json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -207,6 +210,12 @@ public function update_settings() {
 						$updated[ $key ] = $value;
 					};
 					break;
+				case 'jetpack_protect_whitelist':
+					$result = jetpack_protect_save_whitelist( $value, false );
+					if ( ! $result ) {
+						return new WP_Error( 'invalid_ip_address', __( 'One of your IP addresses was not valid.', 'jetpack' ) );
```

Comment:
I don't think we translate Error messages. For instance here we don't https://github.com/Automattic/jetpack/blob/master/json-endpoints/class.wpcom-json-api-get-media-v1-1-endpoint.php